ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -95,7 +114,11 @@ Rectangle {
                 removeAccountConfirmation.active = true
             }
 
-            onHideFromTotalBalanceClicked: RootStore.updateWatchAccountHiddenFromTotalBalance(address, hideFromTotalBalance)
+            onHideFromTotalBalanceClicked: {
+                if (!account)
+                    return
```

Comment:
this condition looks suspicious. Is it possible to click that action when no account is stet?